src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -360,14 +363,18 @@ private void createConference(
             = JitsiMeetGlobalConfig.getGlobalConfig(
                 FocusBundleActivator.bundleContext);
 
-        String id = generateConferenceId();
-        JitsiMeetConferenceImpl conference
-            = new JitsiMeetConferenceImpl(
+        JitsiMeetConferenceImpl conference;
+        String id;
+        synchronized (conferencesSyncRoot)
+        {
+            id = generateConferenceId();
```

Comment:
it could be done as `String id = generateConferenceId();` above the synchronized instead of splitting across the two lines, i mean.